ccd-definition/CaseEvent/User/UserEvents-HWF-R2-CUI-nonprod.json

```diff
@@ -0,0 +1,21 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "PARTIAL_REMISSION_HWF_GRANTED",
+    "Name": "Part remission HwF - granted",
+    "Description": "Part remission HwF granted",
+    "PreConditionState(s)": "PENDING_CASE_ISSUED;HEARING_READINESS",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public",
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/about-to-start",
```

Comment:
i could see in civil-service, about to start event is not using, is this requires in config